### ISSUE TITLE
Failure to grab titles in XO blog

### ISSUE THREAD

**Issue by Marsup** [wallabag/wallabag#2685]
### Issue details

Adding http://blog.eng.xogrp.com/post/154005485319/node-js-promise-enterprise-grade-first-of-all doesn't grab the title. It seems present in some kind of weird way in the initial markup in a \<title\> tag, but I guess it only takes its final place once the javascript is done, so maybe there's that.

### Environment

wallabag.it

### Steps to reproduce/test case

See details ☝️.

**Comment by Kdecherf** [comment]
Fix has been merged into upstream ftr-site-config, it's now @j0k3r's turn to handle that :D

**Comment by j0k3r** [comment]
Thanks @Kdecherf I'll close that one regarding the upcoming wallabag release (which will includean upcoming release of graby-site-config)
